src/content/docs/en/community-resources/content.mdx

```diff
@@ -0,0 +1,84 @@
+---
+title: Community Content
+description: Discover community-produced guides, articles and videos to help you learn and build with Astro!
+i18nReady: false
+---
+
+import Badge from '~/components/Badge.astro';
+import { CardGrid, LinkCard } from '@astrojs/starlight/components';
+import Card from '~/components/Landing/Card.astro'
+import ListCard from '~/components/Landing/ListCard.astro'
+
+PLACEHOLDER TEXT: There's lots more content out there that can help you learn and build with Astro! Here is some content produced and maintained by the Astro community. All links are external and we don't guarantee anything here....
+
+:::tip[Add your own!]
+Have you produced educational content about Astro?
+Open a PR adding a link to this page!
+:::
+## Courses and Tutorials 
+
+Courses and tutorials are focused on teaching you new concepts, often with exercises or sample projects to build. These are a great way to learn new concepts and give you the tools you need for working on your own Astro project.
+
+Check out the following courses and tutorials to learn more about Astro.
+
+  <ListCard title="Introductory Courses" icon="laptop">
+  - [Astro Web Framework Crash Course on freeCodeCamp](https://www.youtube.com/watch?v=e-hTm5VmofI)
+  - [Learn Astro by Chris Pennington <span ><Badge variant="neutral">Paid</Badge></span>](https://learnastro.dev/) 
+  - [Build Modern Websites with Astro by James Q Quick <span><Badge variant="neutral">Paid</Badge></span>](https://astrocourse.dev/) 
+  - [Understanding Astro by Ohans Emmanuel](https://www.ohansemmanuel.com/books/understanding-astro)
+
+  </ListCard>
```

Comment:
I am unsure what it is, but the lack of space between the paragraph and the card here is somewhat jarring. Is there something we can do about this?
<img width="763" alt="image" src="https://github.com/withastro/docs/assets/20650404/bcb937fa-2646-4637-a4ea-3ec68688a7fe">